2.2/Dockerfile

```diff
@@ -14,6 +14,10 @@ LABEL io.k8s.description="Platform for building and running Ruby 2.2 application
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,ruby,ruby22"
 
+ONBUILD ADD . /tmp/src
+ONBUILD RUN $STI_SCRIPTS_PATH/assemble
+ONBUILD CMD $STI_SCRIPTS_PATH/run
```

Comment:
This seems like an interesting idea, I'm all for using this in all s2i images, but was it supposed to be included in this PR? The subject and description does not mention it. I guess it should have had a separate PR.